search_bar.html

```diff
@@ -0,0 +1,49 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
+    <link rel="stylesheet" href="style1.css">
+
+</head>
+<body>
+
+    <form action="https://mate-academy-form-lesson.herokuapp.com/create-application" method="post">
+
+        <label>
+            <div class="search_bar_1 ">
+
+                <img
+                 class="photo"
+                 src="https://raw.githubusercontent.com/mate-academy/layout_search-bar-airbnb/master/src/images/Search.svg"
+                >
+
+                    <input type="search" data-qa="keypress" data-qa="big" class="text" placeholder="Try ''Los Angeles''">  
```

Comment:
```suggestion
                    <input
                      type="search"
                      data-qa="keypress"
                      data-qa="big"
                      class="text"
                      placeholder="Try ''Los Angeles''"
                    >  
```
Поправ скрізь форматування